### PR TITLE
remove search from client and link to external search-client

### PR DESCRIPTION
**Pull request by DallanQ**
Since we now have a new search-client that is separate from client so it can be deployed independently, even deployed to a wordpress page, I have removed the search code from client and made the search menu item in client an external link. 

The question is where should it link to. I could certainly link it to the wordpress page that I created at https://ourroots.org/search2, but it would be nice to deploy the search-client to a separate s3 bucket or maybe better to a subdirectory of the "site" bucket and serve it from there. What do you think? Could we get that to work in the deploy script?

### CODE REVIEW

File: client/.env.production [hunk 1]
```diff
@@ -1,4 +1,5 @@
 VUE_APP_API_BASE_URL=/api
+# VUE_APP_SEARCH_URL=
```
Comment (DallanQ) [inline client/.env.production:2]: Need to decide where the search client lives in production.

File: deploy/awslambda/deploy.sh [hunk 2]
```diff
@@ -19,9 +19,7 @@ aws cloudformation package --template-file "${template_file}" --output-template-
...
 
 # echo Uploading static site content...
-pushd "${scriptdir}/../../client/"
-# This should probably be done in the Makefile
-npm run build
-popd
 aws s3 sync "${scriptdir}/../../client/dist/" "s3://${ENVIRONMENT_NAME}-site/" --delete
+# where should the search client go?
+#aws s3 sync "${scriptdir}/../../search-client/dist/" "s3://${ENVIRONMENT_NAME}-search-site/" --delete
```
Comment (DallanQ) [inline deploy/awslambda/deploy.sh:24]: Could we copy it somewhere as part of the deploy script?

Comment (jancona) [reply deploy/awslambda/deploy.sh:24]: I don't see why not. If you did something like:
```
aws s3 sync "${scriptdir}/../../search-client/dist/" "s3://${ENVIRONMENT_NAME}-site/search/" --delete
```
Assuming that syntax works (I haven't tested putting the subdirectory in the destination URL), you'd have to make sure the path you use didn't conflict with anything in the regular client, but we should be able to serve both of them out of the same bucket.

You'd have to add another route/integration combo in the CF similar to [this](https://github.com/OurRootsOrg/cms-server/blob/master/deploy/awslambda/cms-aurora.cf.yaml#L103-L122). 

Do you want to take a stab at it, or should I?

Comment (DallanQ) [reply deploy/awslambda/deploy.sh:24]: Thanks. I'll take a stab at it. I'm out today but will be back tomorrow.

**Comment by DallanQ** [comment]
@jancona This is ready for your review now.

**Review by jancona — APPROVED** [review]
Looks good!

**Pull request merged**